pages/ecosystem/index.vue

```diff
@@ -10,8 +10,9 @@
           'community projects'
         ]"
       />
+      from Qiskit
       <br>
-      from Qiskit and the Qiskit community
+      and the Qiskit community
     </AppPageHeaderFixed>
     <section id="meet-the-advocates" class="bx--grid meet-the-advocates">
```

Comment:
one comment here:

I know it's not a reported bug, but I'm seeing the `id="meet-the-advocates"`, along with several classes of `meet-the-advocates` throughout the page.

Do you think we can change this to `ecosystem`?

(it _should_ still function the same IMO)